framework/CustomModes/MPISimulationMode.py

```diff
@@ -56,6 +56,13 @@ def modifyInfo(self, runInfoDict):
       @ In, runInfoDict, dict, the original runInfo
       @ Out, newRunInfo, dict, of modified values
     """
+    magicSyntax = False
+    if runInfoDict['NumMPI'] == -runInfoDict['NumThreads']:
+      # our special keyword
+      # we selected the nodes as select=batchSize:ncpus=NumThreads
+      # in the PBS_NODEFILE we have only a batchSize number of nodes
+      # but we want to have NumThreads nodes in each of them
+      magicSyntax = True
```

Comment:
@joshua-cogliati-inl  this is the magic syntax for requiring nodes as select=XX:ncpus=YY

I would appreciate any suggestion on how to improve this ( it is very ugly now)